packages/protocol/contracts/governance/GovernanceSlasher.sol

```diff
@@ -119,7 +83,7 @@ contract GovernanceSlasher is
     address[] calldata electionLessers,
     address[] calldata electionGreaters,
     uint256[] calldata electionIndices
-  ) external onlyL2 onlyAuthorizedToSlash returns (bool) {
+  ) external onlyAuthorizedToSlash returns (bool) {
```

Comment:
Is it a deliberate decision to keep the slashing function's name `slashL2`? For backwards compatibility? Given the amount of deprecations there's going to be in this release, and that this isn't a directly user-facing function, might make sense to just rename this one to `slash`?